### PR TITLE
Get Swift working again

### PR DESCRIPTION
**Pull request by craigtracey**
After a series of refactors and rewrites there were a small number of
defects in the Swift code path. Update Ursula to take account for
these changes.

### CODE REVIEW

File: roles/swift-ring/tasks/main.yml [hunk 1]
```diff
@@ -10,8 +10,6 @@
   when: not should_boostrap|success
 
 - name: setup swift rings
-  command: /usr/local/bin/swifttool -i /home/swiftops/.ssh/id_rsa
+  command: sudo -u swiftops /usr/local/bin/swifttool -i /home/swiftops/.ssh/id_rsa
            bootstrap --config /etc/swift/ring_definition.yml --outdir /etc/swift
-  sudo: yes
-  sudo_user: swiftops
```
Comment (j2sol) [inline roles/swift-ring/tasks/main.yml]: Curious why sudo_user didn't work here.

File: site.yml [hunk 2]
```diff
@@ -207,6 +207,7 @@
       tags: ['openstack', 'horizon', 'control']
 
 - name: swift code and config
+  gather_facts: force
```
Comment (j2sol) [inline site.yml]: This will conflict with an already open PR #941

File: roles/swift-ring/tasks/main.yml (hunk 1 above)
Comment (craigtracey) [inline roles/swift-ring/tasks/main.yml]: No idea, but it doesnt.  Wasnt about to debug it at this point. Sudoers was not happy and looking for a password.

File: site.yml (hunk 2 above)
Comment (craigtracey) [inline site.yml]: An easy rebase

**Comment by craigtracey** [comment]
Waiting on a package build

**Comment by craigtracey** [comment]
Unrelated failure in tests. Rerunning.

**Comment by j2sol** [comment]
Rebased on master and pushed again for tests.

**Pull request merged**